src/Components/Patient/DailyRoundListDetails.tsx

```diff
@@ -11,23 +11,29 @@ const Loading = lazy(() => import("../Common/Loading"));
 const symptomChoices = [...SYMPTOM_CHOICES];
 const currentHealthChoices = [...CURRENT_HEALTH_CHANGE];
 
+function useConsultationQuery(consultationId: any, id: any) {
+  return useQuery(routes.getDailyReport, {
+    pathParams: {
+      consultationId,
+      id,
+    },
+  });
+}
+
 export const DailyRoundListDetails = (props: any) => {
   const { facilityId, patientId, consultationId, id } = props;
-  const dispatch: any = useDispatch();
   const [dailyRoundListDetailsData, setDailyRoundListDetails] =
     useState<DailyRoundsModel>({});
   const [isLoading, setIsLoading] = useState(false);
 
+  const res = useConsultationQuery(consultationId, id);
   const fetchpatient = useCallback(
     async (status: statusType) => {
       setIsLoading(true);
```

Comment:
`useCallback` is no longer needed since `useQuery` handles the fetch for you on mount and whenever the search params / request body variable passed to it changes.